engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/JobPreparationActor.scala

```diff
@@ -11,6 +11,7 @@ import wdl4s.expression.WdlStandardLibraryFunctions
 import wdl4s.util.TryUtil
 import wdl4s.values.WdlValue
 
+import scala.concurrent.ExecutionContext
```

Comment:
This class isn't spawning separate `Future`s, so this import and the implicit below shouldn't be needed.